README.md

```diff
@@ -1,3 +1,5 @@
+<!-- markdownlint-disable MD033 MD013 -->
```

Comment:
why do we need to disable these rules?

In this change we don't use inline html ([md033](https://github.com/DavidAnson/markdownlint/blob/main/doc/md033.md)) also the line lenght ([md013](https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md)) can be solved by distributing the text a bit